examples/step-57/doc/intro.dox

```diff
@@ -0,0 +1,336 @@
+                \end{array}
+              \right)\\
+              \\
+  &=& \left(
+                \begin{array}{c}
+                  - \nu\Delta\delta \mathbf{u}^{k} + \mathbf{u}^{k}\cdot\nabla\delta\mathbf{u}^{k}+\delta\mathbf{u}^{k}\cdot\nabla\mathbf{u}^{k}+ \nabla\delta p^{k}\\
+                  - \nabla \cdot\delta \mathbf{u}^{k}\\
+                \end{array}
+              \right).
+@f}
+Then we have a linearized system
+
+@f{eqnarray*}
+- \nu\Delta\delta\mathbf{u}^{k} + \mathbf{u}^{k}\cdot\nabla\delta\mathbf{u}^{k}+\delta\mathbf{u}^{k}\cdot\nabla\mathbf{u}^{k}+ \nabla\delta p^{k} = \mathbf{g}, \\
+- \nabla \cdot\delta \mathbf{u}^{k} = \nabla\cdot\mathbf{u}^{k},
+@f}
+
+where $\textbf{g} =\textbf{f}+\nu \Delta\textbf{u}^k -(\textbf{u}^k
+\cdot \nabla)\textbf{u}^k -\nabla p^k$, $\textbf{f}$ is the right hand
+side of NSE, $\textbf{u}^k$ and $p^k$ are the solutions from the
+previous iteration. In this linear system, the only unknowns are the
+update terms: $\delta \textbf{u}^{k}$ and $\delta p^{k}$, then the
+strategy we use in step-22 can be used to solve it. The weak form is
+derived like it is done in step-22.
+
+Now, Newton's iteration for solving the NSE is:
+
+<ol>
+  <li> Initialization: Initial guess $u_0$ and $p_0$, tolerance $\tau$;
+  <li> Seek for update term: solve for $\delta\textbf{u}^{k}$ and $\delta p^k$;
+  <li> Update the approximation: $\textbf{u}^{k+1} = \textbf{u}^{k} + \delta\textbf{u}^{k}$ and $p^{k+1} = p^{k} + \delta p^{k}$;
+  <li> Check residual norm: $E^{k+1} = \|F(\mathbf{u}^{k+1}, p^{k+1})\|$. 
+       If $E^{k+1} \leq \tau$, STOP. 
+       If $E^{k+1} > \tau$, back to step 2. 
+</ol>
+
+<h3> Finding an Initial Guess </h3>
+
+For Newton's method to converge, the initial guess needs to be close
+enough to the solution, so it is crucial to find a good starting value.
+
+While the viscosity $\nu$ is large, 1/400 in test case 1, we set the
+initial guess to be the solution to the Stokes equation with viscosity
+$\nu$. For what value of $\nu$ is large, this can only be determined via numerical experiment. In this tutorial, we accept that the solution to Stokes problem is a 
+good initial guess if the visociy of NSE is 1/1000, and this is what we do at the first step while finding an initial guess if viscosity is small. 
+
+However, it convective term $(\mathbf{u}\cdot\nabla)\mathbf{u}$
+gets to be donimant if viscosity is small, like 1/10000 in test case 2. 
+In this situation, we have to set up a series of auxiliary NSE with
+viscosity aprroaching to the one in the target NSE. Correspondingly, we set a sequence
+$\{\nu_{i}\}$ with $\nu_{n}= \nu$, and accept that the solutions to two 
+NSE with viscosity $\nu_{i}$ and $\nu_{i+1}$ are closed if $|\nu_{i} - \nu_{i+1}|$ is small. 
+Then we use the solution to the NSE with viscosity $\nu_{i}$ as the initial guess of the NSE
+with $\nu_{i+1}$. It is like a staircase from Stokes equations to the
+NSE we want to solve. 

```

Comment:
I guess you want to mention here that this is a simple continuation method. And a more general statement would also be appropriate, namely that no stationary solution to the Navier-Stokes exists as soon as the Reynolds number exceeds a critical value because the flow gets turbulent (and thus, intrinsically time-dependent).